app/views/submissions/_diff_viewer.html.erb

```diff
@@ -0,0 +1,31 @@
+<div id="diff-box">
+  <% if @prevVersion %>
+    <% title = "Diff with previous version of #{@displayFilename} (version #{@prevVersion[:version]})" %>
+    <% diff_header = "--- #{title}\n+++ #{title}\n" %>
+    <% begin %>
+    <% diff_str = Diffy::Diff.new(@prev_file_contents, @file_contents, include_diff_info: true).to_s(:text) %>
+    <% if diff_str.present? %>
```

Comment:
small nit but maybe we could condense this

<% diff_str = diff_str.present? ? [diff_header] + diff_str.lines.slice(2..-1)&.join("") : diff_header %>